hack/aks/Makefile

```diff
@@ -8,13 +8,15 @@ AZIMG   = mcr.microsoft.com/azure-cli
 AZCLI   ?= docker run --rm -v $(AZCFG):/root/.azure -v $(KUBECFG):/root/.kube -v $(SSH):/root/.ssh -v $(PWD):/root/tmpsrc $(AZIMG) az
 
 # overrideable defaults
-REGION          ?= westus2
+AUTOUPGRADE     ?= patch
+K8S_VER         ?= 1.25 # Used only for ubuntu 18 as K8S 1.24.9, as K8S > 1.25 have Ubuntu 22
+NODE_COUNT      ?= 2
+NODEUPGRADE     ?= NodeImage
 OS_SKU          ?= Ubuntu
-WINDOWS_OS_SKU  ?= Windows2022
+OS_SKU_WIN      ?= Windows2022
+REGION          ?= westus2
 VM_SIZE	        ?= Standard_B2s
-NODE_COUNT      ?= 2
```

Comment:
Figured `NODE_COUNT` and `REGION` were reasonable defaults to have. Any reason for removal here?